integration/templates/next-app-router/package.json

```diff
@@ -12,7 +12,7 @@
     "@types/node": "^18.17.0",
     "@types/react": "18.2.48",
     "@types/react-dom": "18.2.18",
-    "next": "13",
+    "next": "13.5.4",
```

Comment:
I think we should keep the `13` version here, as this is about integration testing and we want the latest 13 version of next to be installed.

Reply:
You are correct, it shouldn't be explicit. Is it okay if i put something like this `^13.5`?
This change is applied to resolve this: https://github.com/clerk/javascript/security/dependabot/234

Reply:
Nice catch!